src/camera.cpp

```diff
@@ -71,7 +71,9 @@ Camera::Camera(MapDrawControl &draw_control, Client *client):
 	 */
 	m_cache_fall_bobbing_amount = g_settings->getFloat("fall_bobbing_amount");
 	m_cache_view_bobbing_amount = g_settings->getFloat("view_bobbing_amount");
-	m_cache_fov                 = g_settings->getFloat("fov");
+	// 45 degrees is the lowest FOV that doesn't cause the server to treat this
+	// as a zoom FOV and load world beyond the set server limits.
+	m_cache_fov                 = std::fmax(g_settings->getFloat("fov"), 45.0f);
```

Comment:
The limitation should also be done server-side, so it can't be exploited.

Reply:
Done.